static/js/modules/columns.js

```diff
@@ -268,20 +281,39 @@ var filings = {
     orderable: false,
     render: function(data, type, row) {
       var doc_description = row.document_description ? row.document_description : row.form_type;
+      var show_version = true;
+      var is_original = false;
+      var amendment_num = 1;
       var pdf_url = row.pdf_url ? row.pdf_url : null;
       var csv_url = row.csv_url ? row.csv_url : null;
       var fec_url = row.fec_url ? row.fec_url : null;
 
+      // because of messy data, do not show if not e-filing or null amendment indicator
+      if(row.means_filed !== 'e-file'|| row.amendment_indicator === null) {
+        show_version = false;
+      }
+
+      if (row.amendment_indicator === 'N') {
+        is_original = true;
+      }
+
+      if (row.amendment_chain) {
+        amendment_num = row.amendment_chain.length;
```

Comment:
I think this should be `row.amendment_chain.length - 1`? The numbers seem like they're inflated by 1.